From faa3920f1d398ce01aa960184f5af19c4996fc19 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 7 Mar 2016 15:44:20 +0100
Subject: [PATCH] network/net-genrules.sh: add physical vlan network interfaces

due to the removal of vlan.info, parse all vlan.*.phy files
---
 modules.d/40network/net-genrules.sh | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/modules.d/40network/net-genrules.sh b/modules.d/40network/net-genrules.sh
index fc303f4..ce033d3 100755
--- a/modules.d/40network/net-genrules.sh
+++ b/modules.d/40network/net-genrules.sh
@@ -24,10 +24,10 @@ fix_bootif() {
         IFACES="$IFACES $ethname"
     fi
 
-    if [ -e /tmp/vlan.info ]; then
-        . /tmp/vlan.info
-        IFACES="$IFACES $phydevice"
-    fi
+    for i in /tmp/vlan.*.phy; do
+        [ -e "$i" ] || continue
+        IFACES="$IFACES $(cat "$i")"
+    done
 
     # bond: attempt only the defined interface (override bridge defines)
     if [ -e /tmp/bond.info ]; then
